core/modules/image/image.field.inc

```diff
@@ -607,6 +608,24 @@ function image_field_formatter_settings_form($field, $instance, $view_mode, $for
     ),
   );
 
+  if (!isset($settings['image_load'])) {
+    $settings['image_load'] = 'normal';
```

Comment:
I suppose, this should be "auto" instead of "normal".

Reply:
good catch, thank you!